### PR TITLE
Fix: changes in browse page

### PR DESCRIPTION
**Pull request by SirineMhedhbi**
## Context
 Fix changes in browse page

## Changes

- [x] Change the styling of the submit button on the homepage.
image

- [x]  Change the contrast of the box on the browse page.
image

- [x]  Change the bloc  admin to alert.
image

- [x]  Change the format of date.
image

- [x]  Change "See more " and "See less".
image

- [x]  Change "Generic Types" par "Ontology types" . 
image
- [x] Changer "e.g AGROVOC" to  "e.g., AGROVOC".
image

- [x] Change deprecated button to Retired button.

### CODE REVIEW

File: app/assets/stylesheets/browse.scss [hunk 1]
```diff
@@ -334,8 +335,17 @@
   border-radius: 5px;
   padding: 14px 20px;
   margin-bottom: 12px;
+  box-shadow: 2px 30px 60px rgba(0, 0, 0, 0.05);
+}
 
+.align-items-alert {
```
Comment (syphax-bouazzouni) [inline app/assets/stylesheets/browse.scss:341]: you can use directly bootstrap like this `class="d-flex align-items-center"`, this will prevent us from handling a new CSS class.

Comment (syphax-bouazzouni) [reply app/assets/stylesheets/browse.scss:341]: same for align-alert

File: app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml [hunk 2]
```diff
@@ -8,11 +8,10 @@
             %i.fas.fa-key{title:"Private Ontology"}
           - ontology_status = status_string(ontology)
           = render Display::InfoTooltipComponent.new(text: ontology_status, icon: submission_status_icons(ontology_status)) if session[:user]&.admin?
-      .browse-desc-container{data:{controller:"text-truncate", 'text-truncate-more-text-value': '+ Show more ...' , 'text-truncate-less-text-value': '- Show less ...'}}
+      .browse-desc-container{data:{controller:"text-truncate", 'text-truncate-more-text-value': '...+' , 'text-truncate-less-text-value': '...-'}}
```
Comment (syphax-bouazzouni) [inline app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml:11]: to confirm with @jonquet , it is less visible like this (users will not see it)

File: app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml [hunk 3]
```diff
@@ -35,15 +34,16 @@
...
         - if ontology[:format]
           .browse-ontology-view
             = ontology[:format]
         - if ontology[:deprecated]
           .browse-ontology-deprecated
-            deprecated
+            Retired
         - if ontology[:viewOfOnt]
           .browse-ontology-view{data:{controller:'tooltip'}, title: "View of the ontology #{ontology[:viewOfOnt].split('/').last}"}
             view
```
Comment (syphax-bouazzouni) [inline app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml:49]: Capitalize "view" term, also

File: app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml [hunk 4]
```diff
@@ -35,15 +34,16 @@
                 = ontology[:contact].humanize
 
         - if ontology[:released]
-          .browse-ontology-view{data:{controller:'tooltip'}, title: "Release date #{ontology[:released].split('/').last}"}
-            = DateTime.parse(ontology[:released]).year
+          - date = render DateTimeFieldComponent.new(value: ontology[:released])
```
Comment (syphax-bouazzouni) [inline app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml:37]: good job on this

File: app/components/display/alert_component.rb [hunk 5]
```diff
@@ -1,9 +1,10 @@
 class Display::AlertComponent < ViewComponent::Base
-    def initialize(message: nil, closable: true, type: "info", auto_close_delay: nil)
+    def initialize(message: nil, closable: true, type: "info", auto_close_delay: nil, style: nil)
```
Comment (syphax-bouazzouni) [inline app/components/display/alert_component.rb:2]: you will need to explain to me why this change, and add an argument `style` to just have` justify-content: space-between`. Is there no better way to do this?

File: app/views/ontologies/browser/browse.html.haml [hunk 6]
```diff
@@ -1,16 +1,14 @@
...
-        %div
-          %div.browse-you-are-admin
-            %h3 Welcome admin
-            %div
-              %svg{:fill => "none", :height => "11", :viewbox => "0 0 11 11", :width => "11", :xmlns => "http://www.w3.org/2000/svg"}
-                %circle{:cx => "5.5", :cy => "5.5", :fill => 'var(--admin-color)', :r => "5.5"}
-              %p This coloring indicates admin-only features
-
+        = render Display::AlertComponent.new(type: 'info', style: "justify-content: space-between;") do
+          %strong Welcome admin
```
Comment (syphax-bouazzouni) [inline app/views/ontologies/browser/browse.html.haml:6]: you can create here a div to set the `justify-content: center` instead of adding it into the component? (see comment in the alert component )

File: app/views/ontologies/browser/browse.html.haml [hunk 7]
```diff
@@ -1,16 +1,14 @@
 .browse-center
   .browse-container
-
-    .browse-submit-new-ontology-and-you-are-admin-container
+    .container.align-alert
```
Comment (syphax-bouazzouni) [inline app/views/ontologies/browser/browse.html.haml:3]: the alert can be a little bigger in the width, no?

File: app/views/ontologies/browser/browse.html.haml [hunk 8]
```diff
@@ -77,7 +75,7 @@
         .browse-second-row
           .browse-search-bar
             .browse-search-container
-              %input{:name => "search", :placeholder => "Start typing to filter ontologies, e.g AGROVOC...", :type => "text", :value => @search, data: {action: "input->browse-filters#dispatchInputEvent"}}
+              %input{:name => "search", :placeholder => "Start typing to filter ontologies, e.g., AGROVOC...", :type => "text", :value => @search, data: {action: "input->browse-filters#dispatchInputEvent"}}
```
Comment (syphax-bouazzouni) [inline app/views/ontologies/browser/browse.html.haml:78]: if you can use the localized value, it would be better

**Review by syphax-bouazzouni — CHANGES_REQUESTED** [review]

File: app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml (hunk 2 above)
Comment (SirineMhedhbi) [reply app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml:11]: Yes, that's what I see too

File: app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml (hunk 3 above)
Comment (SirineMhedhbi) [reply app/components/ontology_browse_card_component/ontology_browse_card_component.html.haml:49]: okey

File: app/components/display/alert_component.rb (hunk 5 above)
Comment (SirineMhedhbi) [reply app/components/display/alert_component.rb:2]: I tried with other methods, but it didn't work, so that's why I used this method to add a style to the component

File: app/views/ontologies/browser/browse.html.haml (hunk 7 above)
Comment (SirineMhedhbi) [reply app/views/ontologies/browser/browse.html.haml:3]: I tried to do it with the same width as the old block =>  to confirm with @jonquet

File: app/views/ontologies/browser/browse.html.haml (hunk 8 above)
Comment (SirineMhedhbi) [reply app/views/ontologies/browser/browse.html.haml:78]: I couldn't find the localized value in the development branch, but i will try to update it. ;)

File: app/assets/stylesheets/browse.scss [hunk 9]
```diff
@@ -100,6 +100,7 @@
   display: flex;
   justify-content: space-between;
   position: relative;
+  box-shadow: 2px 30px 60px rgba(0, 0, 0, 0.05);
```
Comment (Bilelkihal) [inline app/assets/stylesheets/browse.scss:103]: I don't see the point of adding this shadow, it was clear and going well with the whole style, do you have any justification for doing this?

Comment (syphax-bouazzouni) [reply app/assets/stylesheets/browse.scss]: The shadow was to have a better distinction between the blocks(cards), and in Mac screens, it was too bright.

**Comment by syphax-bouazzouni** [comment]
FYI @SirineMhedhbi , I did the following changes: 
* make the box-shadow of the browse page lighter (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/363/commits/e1e478464d6429301c2d6c926b40c755c9ac2c51)
* make the upload button in the browse page take all the width (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/363/commits/bf22353575730b8b5ac2f190c5dcf9b2f3e16bcd)
* use the same browser page upload button component in the home page (https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/363/commits/d55fe9482e026394dacb0da72b9d703e0b8ad7de)

File: app/views/ontologies/browser/browse.html.haml (hunk 6 above)
Comment (SirineMhedhbi) [reply app/views/ontologies/browser/browse.html.haml]: I change it 
image

File: app/views/ontologies/browser/browse.html.haml (hunk 8 above)
Comment (syphax-bouazzouni) [reply app/views/ontologies/browser/browse.html.haml:78]: Todo in an other iteration. @SirineMhedhbi

**Review by syphax-bouazzouni — APPROVED** [review]

**Comment by syphax-bouazzouni** [comment]
Seems good to me, thanks @SirineMhedhbi. I merged it and let you open another pull request with the same branch if needed.
